lerna.json

```diff
@@ -2,7 +2,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "0.0.7",
+  "version": "independent",
```

Comment:
Alex, any specific use case  why we are adopting independent instead of maintaining the same version number across all packages. Here are the pros and cons that i felt if we use the default standard

Pros:
- All packages follow the same major version, so when they are being added to project independently we will get to know easily if a specific package with version 2 goes well with our package version 
Cons:
- One major con that i see with this approach is, if we are introducing a breaking change to a specific package and want all the end users to upgrade, then we need to intentionally bump the version number of all packages :grin: